nettests/ts-026-riseupvpn.md

```diff
@@ -9,47 +9,46 @@ RiseupVPN
 # Test preconditions
 
 * An internet connection with functional DNS
-* When testing the connection to a riseupvpn gateway, we assume that OpenVPN is already installed
 
 # Expected impact
 
-The ability to detect both if the (RiseupVPN) API of the provider and its gateways can be reached and connected to. TCP-connection and HTTP GET request provide a JSON-file which contains the IP addresses of the VPN gateways. A second request to the API will hand out the CA-material to verify the gateways. Different transports will be tested, depending on their capabilities as described by the provider (TCP, UDP, ports, OpenVPN, obfs4,...).
-
+The ability to detect both if the (RiseupVPN) API of the provider and its gateways can be reached and connected to. 
 # Expected inputs
 
 ## Import document or import data format
 None. In a next iteration you can specify a provider, the default is riseup.net
 
-## Semantics
-
-DO WE WANT THIS, DO WE WANT THIS NOW, LIKE THIS? FORMAT REQUIRES IMPROVEMENT
-* A provider can be specified with their domain name: https://calyx.net
-* A specific gateway and transport can be tested with a url: ovpn://192.168.1.1:1194
-
 # Test description
 
-This test will check if a VPN service like RiseupVPN is working as exepected. If the first fails, we can't execute the second. When the probe has OpenVPN installed, we can execute the second test, otherwise we only do the first.
+This test will check if a LEAP platform based VPN service like RiseupVPN is working as exepected. The experiment consists of two parts.
 
 1. The RiseupVPN API service
-1. The advertised gateways (the VPN servers used by RiseupVPN clients)
+2. The advertised gateways (the VPN servers used by RiseupVPN clients)
+
 
 ## RiseupVPN API check
 
-The RiseupVPN provider serves a JSON describing which servers a client could use (https://example.org/provider.json). When parsing, we learn where their CA-certificate is stored. The locations of the VPN gateway servers are also described. At last we will request a client certificate, enabling us to proceed to the next tests. This all implies a working DNS.
+A HTTP GET request to a commercial CA certified API endpoint will hand out the self-signed certificate needed to access the provider API. While RiseupVPN clients will perform a fingerprint verification, this test skips that step in order to reduce the complexity.
```

Comment:
Can't think of a better description as 'commercial CA,' but if you have one (Mozilla? Defaults in ...?), we should change it. If we don't let's just keep this. Or maybe just write 'GET request to an API endpoint which will hand out…' ?

Reply:
> Or maybe just write 'GET request to an API endpoint which will hand out…' ?

sounds good